virttest/qemu_storage.py

```diff
@@ -11,6 +11,7 @@
 import os
 import re
 import string
+from distutils import sysconfig
```

Comment:
what about using try except, e.g. 
```
try:
    from collections.abc import Mapping
except ImportError:
    from collections import Mapping
```
Check BytesIO in utils_misc.py for an example

BTW, we cannot compare string for the version, e.g.
```
>>> '3.6' < '3.10'
False
```

But we can use tuple comparison, e.g. 
```
sys.version_info < (3, 10)

>>> (3, 6) < (3, 10)
True
```

Reply:
Yes, agree with you, your method makes more sense. I have updated the fix. Thanks.